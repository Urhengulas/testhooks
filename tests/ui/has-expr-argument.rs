use testhooks::testhooks;

#[testhooks(true)]
mod tests {}

fn main() {}
