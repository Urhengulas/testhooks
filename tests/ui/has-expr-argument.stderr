error: this attribute takes no arguments

  = help: use `#[testhooks]`

 --> $DIR/has-expr-argument.rs:3:1
  |
3 | #[testhooks(true)]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `testhooks` (in Nightly builds, run with -Z macro-backtrace for more info)
